web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -507,6 +507,8 @@
 
         return self;
       };
+	  
+      self['catch'] = self.fail;
```

Comment:
Reserved words [only apply to identifiers](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Reserved_word_usage), so `self.catch = self.fail` should work just fine I believe. Worth double-checking in IE8 and maybe 7 though.